modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+:FeatureName: LSO automatic discovery and provisioning
+
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+Use the following procedure to automatically discover local devices, and to automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name you want to open. The "Node Details" page is displayed.
+
+.. From the *Actions* drop-down list, select *Discover nodes* to allow automatic discovery of the selected  devices on the node.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list now updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
+
+. To automatically provision local volumes for the discovered devices from the web console:
+
+.. Navigate to *Operators* -> *Installed Operators* and select *Local Storage* from the list of Operators.
+
+.. Select *Local Volume Set* -> *Create Local Volume Set*.
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
+
+.. Select the disk type, mode, size, and limit you want to apply to the local volume set, and click *Create*.
++
+A message displays after several minutes, indicating that the "Operator reconciled successfully."
+
```

Comment:
What message will display here?

Reply:
There are actually two messages, one about the DaemonSet and one that the Operator reconciled successfully. I was going to print both but decided to simplify to just the latter message. Do you think I need more details here?

Reply:
aha..got your point. It is better to tell where is the message. :-)

Reply:
I keep losing my cluster access, so forget what the name of the section head is that this message falls under 😝 . Can you tell?